File: 1-Iniciante/ex1036.js
// entrada
let input = require('fs').readFileSync('dev/stdin', 'utf-8'); //adicionar uma barra antes do 'dev' para funcionar no beecrowd
let [a, b, c] = input.split(' ').map(n => parseFloat(n));

// processamento & saida
if(a === 0.0){
    console.log('Impossivel calcular');
}
else{
    let delta = b ** 2 - (4 * a * c);
    if(delta < 0.0){
        console.log('Impossivel calcular');
    }
    else{
        let raiz1 = (-b + Math.sqrt(delta)) / (2 * a);
        let raiz2 = (-b - Math.sqrt(delta)) / (2 * a);
        console.log(`R1 = ${raiz1.toFixed(5)}`);
        console.log(`R2 = ${raiz2.toFixed(5)}`);
    }
}

File: 1-Iniciante/ex1036.py
from math import sqrt
#entrada
a, b, c = str(input()).split()
a = float(a)
b = float(b)
c = float(c)

#processamento
delta = b**2 - 4 * a * c

if delta < 0:
    print("Impossivel calcular")
elif a == 0:
    print("Impossivel calcular")
else:
    x1 = (-b + sqrt(delta)) / (2 * a)
    x2 = (-b - sqrt(delta)) / (2 * a)

    print("R1 = {:.5f}".format(x1))
    print("R2 = {:.5f}".format(x2))
